### PR TITLE
Update crate metadata

### LINKED ISSUE #1030: meta: Cargo.toml package info seems out-of-date
I don't know if this is intentional, but the name of the `docs.rs` crate is still `cratesfyi` and the author is Onur Aslan:

https://github.com/rust-lang/docs.rs/blob/49099068e383bacb67f775537d3e61a3f58919e0/Cargo.toml#L1-L4

Shouldn't the name be `docs-rs` or something and the author be `The Rust Project Developers` like with the rustc crates?

### PR DESCRIPTION
**Pull request by camelid**
Closes #1030.

Added "The Rust Project Developers" to the end of the "author" field in
the `Cargo.toml` metadata for each crate that makes up docs.rs.

Also changed the name of the root crate from "cratesfyi" to "docs-rs", but I left the binary's name alone due to @jyn514's [concerns](https://github.com/rust-lang/docs.rs/issues/1030#issuecomment-686172823) about having to update all the production scripts to use the new name.

### CODE REVIEW

**Comment by jyn514** [comment]
Hmm, I was imagining this would change the name of the crate without renaming the binary. It's fine if you don't do it in this PR, but then maybe don't say it will close #1030?

**Comment by camelid** [comment]
Oh, I misunderstand your comment (https://github.com/rust-lang/docs.rs/issues/1030#issuecomment-686172823)! I think I can do that by changing the `Cargo.toml` to this, right?

```toml
# ...
name = "docs-rs"
# ...
[[bin]]
name = "cratesfyi"
```

**Comment by camelid** [comment]
Okay, I changed the crate name (and left the binary name alone)!

**Comment by camelid** [comment]
Hmm, the Docker build failed, even though I didn't update the binary's name.

**Comment by camelid** [comment]
Oops, I forgot to update the usages of the old name...

**Comment by Nemo157** [comment]
> 
> 
> Hmm, the Docker build failed, even though I didn't update the binary's name.

Try changing

https://github.com/rust-lang/docs.rs/blob/bfa6adaabd3081be8e5d0f93d498b93d167469b9/dockerfiles/Dockerfile#L35

to 

```
RUN echo "fn main() {}" > src/bin/cratesfyi.rs && \ 
```

**Comment by camelid** [comment]
@Nemo157 The Docker build is still failing even after that change (7e222e3).

What should I do?

**Comment by Nemo157** [comment]
Oh, stick a `mkdir src/bin &&` just before the `echo`

**Comment by camelid** [comment]
By the way, why does the Dockerfile need to be changed?

**Comment by Nemo157** [comment]
It was previously relying on the fact that the "default" binary at `src/main.rs` was also called `cratesfyi`, and using that to prebuild all the dependencies before copying the source in and building the actual binary, so that changing the source code and rebuilding the image doesn't need to rebuild all the dependencies (for local testing in docker).

**Comment by camelid** [comment]
Okay, CI is finally passing. There were a bunch more spots where "cratesfyi" was used, but I was worried that changing them would require changing those production scripts that @jyn514 was worried about.

**Review by jyn514 — APPROVED** [review]
r=me with comment suggestion applied

File: src/db/file.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //! Simple module to store files in database.
 //!
-//! cratesfyi is generating more than 5 million files, they are small and mostly html files.
+//! docs.rs is generating more than 5 million files, they are small and mostly html files.
 //! They are using so many inodes and it is better to store them in database instead of
 //! filesystem. This module is adding files into database and retrieving them.
```
[the remark this answers is not in the document]
Comment (camelid) [reply src/db/file.rs:5]: Sure! :)

**Comment by jyn514** [comment]
Oh one last thing - please change `DOCSRS_LOG=cratesfyi,rustwide=info` to `DOCSRS_LOG=docs_rs,rustwide=info` in `.env.sample`.

cc @rust-lang/docs-rs , you'll need to do the same in `.env`.

**Comment by camelid** [comment]
Updated!

**Comment by jyn514** [comment]
Thanks for sticking with this!

**Comment by Nemo157** [comment]
Mmm, would that rather be `DOCSRS_LOG=cratesfyi,docs_rs,rustwide=info` to catch logs from the binary too?

**Comment by jyn514** [comment]
Ah hmm, yeah I think you're right. Does the binary even do any logging though? I thought it just printed to stdout on error.

**Comment by Nemo157** [comment]
There's a single log line in it:

https://github.com/rust-lang/docs.rs/blob/b478d4d93d865f4dc05e802677538b5d8b908074/src/bin/cratesfyi.rs#L125

**Comment by jyn514** [comment]
maybe the fix should be to remove the --foreground option :P

**Pull request merged**